upload.sh

```diff
@@ -607,10 +607,11 @@ _setup_arguments() {
                 FOLDERNAME="${OPTARG}"
                 ;;
             r)
-                _check_default "${OPTARG}" "ROOTDIR=${OPTARG/default=/}" "UPDATE_DEFAULT_ROOTDIR=_update_config"
+                ROOTDIR="${OPTARG/default=/}"
+                [[ ${OPTARG} = default* ]] && UPDATE_DEFAULT_ROOTDIR="_update_config" || :
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Note that A && B || C is not if-then-else. C may run when A is true.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5565059)